src/ticket/Ticket.java

```diff
@@ -0,0 +1,28 @@
+package ticket;
+
+import datetimeutils.DateTimeUtils;
+
+public class Ticket {
+
+    public long fineID;
+    private final GovAuthorities authorityName;
+    public DateTimeUtils issueDate;
+    public Boolean isPaid;
+    public double fine;
+    private static final double UMA = 207.44;
+
+    public Ticket(long fineID, DateTimeUtils issueDate, GovAuthorities authorityName, Boolean isPaid) {
+        this.fineID = fineID;
+        this.issueDate = issueDate;
```

Comment:
Al ser el constructor para crear un ticket nuevo por default yo le pondría la fecha actual. por ejemplo
```suggestion
        this.issueDate = DateTimeUtils. getCurrentZonedDateTime();
```

De esta manera la fecha se settea al crear el objeto y evitas problemas.

Por ejemplo:
Imagina que algún programador ponga una fecha de hace 10 años por error, entonces tu registro estará erroneo